README.md

```diff
@@ -10,10 +12,30 @@
 
 4. Are there unexpected regions with higher stars (rare patterns)? Something non-trivial that may be discovered are "hidden gems" such as food cart street or local spots
 
+5. Are more active yelpers prone to giving higher ratings or more thorough reviews?
```

Comment:
"thorough reviews" could be interesting to look into.  Would be interesting to see if longer text reviews have any relationship with more extreme reviews good/bad

Reply:
I agree. As we process the reviews, we may find some interesting relationships, such as star rating and word count or word frequency. Perhaps we can merge this with (2) "What are frequent features of 1-star.. and 5-star.." since we may find that 1-star reviews do indeed have a common feature of being shorter and less detailed.

Reply:
I have a feeling there are two main "long" restaurant reviews. People ranting about how terrible a place is - poor rating, and foodies giving their detailed review which is not glowing....still has more than 1 star because they like the industry.